docs/source/examples/managed-jobs.rst

```diff
@@ -179,17 +302,74 @@ Below is an example of mounting a bucket to :code:`/checkpoint`.
       mode: MOUNT
 
 The :code:`MOUNT` mode in :ref:`SkyPilot bucket mounting <sky-storage>` ensures the checkpoints outputted to :code:`/checkpoint` are automatically synced to a persistent bucket.
-Note that the application code should save program checkpoints periodically and reload those states when the job is restarted.
-This is typically achieved by reloading the latest checkpoint at the beginning of your program.
+
+To implement checkpointing in your application code:
+
+1. Periodically save checkpoints containing the current program state during execution.
+2. When starting/restarting, check if any checkpoints are present, and load the latest checkpoint.
+
+Checkpointing features are included in most model training libraries, such as `PyTorch <https://pytorch.org/docs/stable/checkpoint.html>` and `TensorFlow <https://www.tensorflow.org/guide/checkpoint>`.
```

Comment:
Is this text reasonable? I don't really know what I'm talking about when it comes to ML libraries.

Reply:
This is correct :-) Suggested.